### ISSUE TITLE
WCAG 1.1.1: Images must have alternate text (img[src$="pair\.png"])

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#2291]
#### Issue
`Images must have alternate text` ([image-alt](https://dequeuniversity.com/rules/axe/3.2/image-alt?application=msftAI))
#### Target application
[Microsoft MakeCode for micro:bit](https://makecode.microbit.org/#editor)
#### Element path
img[src$="pair\.png"]
#### Snippet
`<img class="ui medium rounded image" src="./static/download/pair.png">`
#### How to fix
    Fix any of the following:
      Element does not have an alt attribute
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute or the title attribute is empty
      Element's default semantics were not overridden with role="presentation"
      Element's default semantics were not overridden with role="none"
#### Environment
Chrome version 75.0.3770.142
This accessibility issue was found using Accessibility Insights for Web 2.7.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

images in the pair device screen

![Screen Shot 2019-07-18 at 2 37 07 PM](https://user-images.githubusercontent.com/5615930/61493890-90f2c400-a969-11e9-96b0-31b44546b870.png)

**Comment by BhuwanNaswa** [comment]
Labels
#A11yMAS, #HCL, #HCL-MakeCode-E2E, #MAS1.1.1, #Win10-Chrome, #Win10-Edge, #accessibility, #K4W, #closed
